assets/sass/components/global/_googlesitekit-publisher-wins.scss

```diff
@@ -20,13 +20,29 @@
 
 	@include googlesitekit-inner-padding;
 	background: $c-base;
+	display: flex;
+	flex-direction: column;
 	opacity: 1;
 	position: relative;
 	transition:
 		opacity $t-default ease-in-out,
 		visibility $t-default ease-in-out;
 	visibility: visible;
 
+	@media (min-width: $bp-tablet) {
+		flex-direction: row;
+
+		.googlesitekit-publisher-win__secondary-pane-divider {
+			border-left: 1px solid $c-athens-gray;
+			height: 106px;
+			margin: auto 30px;
+		}
```

Comment:
I originally tried using a psuedo `::before` selector for the second grid element. But I couldn't easily style the partial border (since the divider doesn't span the full height of the secondary pane) and give it appropriate margins so that it is vertically centred. Need to position things absolutely or use gradients. These three lines for a separate divider div seemed more intuitive and minimal.

Reply:
Ahhh, okay cool. Makes sense, just checking 😄